tests/test_logger.py

```diff
@@ -0,0 +1,43 @@
+import io
+import logging
+import pytest
+
+from datadog_lambda.logger import initialize_logging
+
+_test_initialize_logging = (
```

Comment:
great parameterizing!

Reply:
in python, it's spelled `parametrizing`, whoever wrote it forgot the second `e` 😆

Reply:
I wonder which one is the english version and which is the american one lol

Reply:
According to wikipedia I guess both spells are correct? And that they're all british? https://en.wiktionary.org/wiki/parametrize